disk/disk.go

```diff
@@ -154,6 +154,15 @@ func (d *Disk) Remove(name string, all bool) error {
 	return os.RemoveAll(name)
 }
 
+func (d *Disk) Sync(name string) error {
```

Comment:
I see, I guess I find the need to sync multiple files a bit odd. In the end of the day, you'll still need to fsync one fd at a time so perhaps we usually need to group syncs in case of transactions or some sort, in that case it's like a group commit of a bunch of files with shadowing paging or something similiar.